build-aux/Jenkinsfile.full

```diff
@@ -22,7 +22,7 @@ cd build
 tar -xf ${WORKSPACE}/apache-couchdb-*.tar.gz
 cd apache-couchdb-*
 ./configure --with-curl --spidermonkey-version ${sm_ver}
-make check || (build-aux/logfile-uploader.py && false)
+make check || (make build-reports && false)
```

Comment:
the target is `build-report` not `build-reports`, right?

Reply:
Derp. Fixed.